python/src/aiconfig/editor/server/server.py

```diff
@@ -457,3 +457,30 @@ def set_description() -> FlaskResponse:
     operation = make_op_run_method(MethodName("set_description"))
     operation_args: Result[OpArgs, str] = result.Ok(OpArgs({"description": description}))
     return run_aiconfig_operation_with_op_args(aiconfig, "set_description", operation, operation_args)
+
+@app.route("/api/clear_outputs", methods=["POST"])
```

Comment:
To extend, we can take in list of prompt_names, and if none specified, default to all of them (like what we have below)

Reply:
That would be handy for clearing single outputs later, but can be done at that point, I think